$NetBSD: patch-configure,v 1.2 2013/10/28 14:21:27 richard Exp $

* fix non-portable 'echo -n' utilisation by using GNU configure's 
* `as_echo_n' command.

--- configure.orig	2009-10-19 16:02:17.000000000 +0000
+++ configure
@@ -13740,7 +13740,7 @@ LIBBONOBO_IDL_DIR="`$PKG_CONFIG --variab
 
 
 
-LANGDIRS="`$GREP -v "^\#" po/LINGUAS | while read line; do echo -n "$line/ "; done `"
+LANGDIRS="`$GREP -v "^\#" po/LINGUAS | while read line; do $as_echo_n "$line/ "; done `"
 
 
 
